### ISSUE TITLE
Verify compliance with NeurIPS 2021 D&B formatting requirements

### ISSUE THREAD

**Issue by judithfan** [cogtoolslab/physics-benchmarking-neurips2021#24]
Link to details: https://neurips.cc/Conferences/2021/CallForDatasetsBenchmarks

### Main submission
- [x] Verify that submission is <= 9 content pages in NeurIPS format, including all figures and tables; additional pages containing the paper checklist and references are allowed.
```
Please carefully follow the Latex template for this track when preparing proposals. We follow the NeurIPS format, but with the appropriate headings, and without hiding the names of the authors.

neurips_data_2021.tex -- LaTeX Template

neurips_data_2021.sty -- style file for LaTeX 2e

neurips_data_2021.pdf -- example PDF output
```

- [x] Papers should be submitted via OpenReview (click to start your submission). Reviewing is single-blind, hence the paper should not be anonymized.
- [x] Include public link to the dataset or benchmark data. Set S3 ACL settings to public-read now. @judithfan 

### Supplementary materials

- [x] Dataset documentation and intended uses. Recommended documentation frameworks include datasheets for datasets, dataset nutrition labels, data statements for NLP, and accountability frameworks.@yamins81 / @judithfan 
- [x] Create URL to website/platform where the dataset/benchmark can be viewed and downloaded by the reviewers.  If the dataset can only be released later, you must include instructions for reviewers on how to access the dataset. This can only be done after the first submission: after submission, there will be an 'add dataset or benchmark' button where you can leave information for reviewers. We highly recommend making the dataset publicly available immediately or before the start of the NeurIPS conference. In select cases, requiring solid motivation, the release date can be stretched up to a year after the submission deadline. @judithfan 
    - [ ] Link to model URLs: appendix.tex see section A.2: Computational details
    - [x] Training data generation
    - [x] Test data hdf5s - uploading the hdf5s 
    - [x] Test data mp4 - write script that fetches the "new data" (videos)
    - [x] Human psychophysics - pointer to repo & analysis notebooks/scripts
- [x] Add Author statement that they bear all responsibility in case of violation of rights, etc., and confirmation of the data license. @yamins81 
- [x] Hosting, licensing, and maintenance plan. The choice of hosting platform is yours, as long as you ensure access to the data (possibly through a curated interface) and will provide the necessary maintenance. @judithfan 
To ensure accessibility, we largely follow the NeurIPS guidelines for data submission, but also allowing more freedom for non static datasets. The supplementary materials for datasets must include the following:
- [x] Links to access the dataset and its metadata. This can be hidden upon submission if the dataset is not yet publicly available but must be added in the camera-ready version. In select cases, e.g when the data can only be released at a later date, this can be added afterward. Simulation environments should link to (open source) code repositories. @judithfan 
- [x] The dataset itself should ideally use an open and widely used data format. Provide a detailed explanation on how the dataset can be read. For simulation environments, use existing frameworks or explain how they can be used.
- [x] Long-term preservation: It must be clear that the dataset will be available for a long time, either by uploading to a data repository or by explaining how the authors themselves will ensure this
- [x] Explicit license: Authors must choose a license, ideally a CC license for datasets, or an open source license for code (e.g. RL environments). An overview of licenses can be found here: https://paperswithcode.com/datasets/license @yamins81 
- [x] Add structured metadata to a dataset's meta-data page using Web standards (like schema.org and DCAT): This allows it to be discovered and organized by anyone. A guide can be found here: https://developers.google.com/search/docs/data-types/dataset. If you use an existing data repository, this is often done automatically. @yamins81 
- [x] Highly recommended: a persistent dereferenceable identifier (e.g. a DOI minted by a data repository or a prefix on identifiers.org) for datasets, or a code repository (e.g. GitHub, GitLab,...) for code. If this is not possible or useful, please explain why. @yamins81 
- [x] For benchmarks, the supplementary materials must ensure that all results are easily reproducible. Where possible, use a reproducibility framework such as the ML reproducibility checklist, or otherwise guarantee that all results can be easily reproduced, i.e. all necessary datasets, code, and evaluation procedures must be accessible and documented. @judithfan
